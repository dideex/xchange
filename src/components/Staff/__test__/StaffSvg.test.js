import React from 'react'
import Component from '../StaffSvg'
import {shallow} from 'enzyme'

describe('Staff svg', () => {
  it('Basic staff 1', () => {
    const wrapper = shallow(<Component id="staff1" />)

    expect(wrapper.html()).toMatchSnapshot()
  })
  it('Basic staff 2', () => {
    const wrapper = shallow(<Component id="staff2" />)

    expect(wrapper.html()).toMatchSnapshot()
  })
  it('Empty', () => {
    const wrapper = shallow(<Component id={null} />)

    expect(wrapper.html()).toMatchSnapshot()
  })
})
