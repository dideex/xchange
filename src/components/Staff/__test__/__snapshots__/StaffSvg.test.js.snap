// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Staff svg Basic staff 1 1`] = `"<svg viewBox=\\"0 0 291 167\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M0.050293 166.311C13.0269 70.3545 0.050293 52.4758 84.9063 39.4629C125.755 48.968 164.572 49.3075 201.132 39.4629C308.556 55.8705 270.867 86.0832 290.05 166.311H0.050293Z\\" fill=\\"#211C1D\\"></path><path d=\\"M86.2604 39.4624L110.86 24.4126C133.766 53.3806 156.109 55.0779 177.887 24.4126L202.486 39.4624C205.871 50.8911 206.323 60.3962 202.486 66.8461L188.155 74.4276C193.91 80.4249 206.097 85.4037 202.486 89.4773C198.875 93.551 143.696 147.753 143.696 147.753C143.696 147.753 88.9686 94.5694 86.2604 90.8352C83.5522 87.1011 102.622 72.9566 102.622 72.9566C102.622 72.9566 90.9997 66.054 86.2604 61.9804C81.4082 58.02 86.2604 39.4624 86.2604 39.4624Z\\" fill=\\"#4D4B4C\\"></path><path d=\\"M115.26 33.5784V0.310547H174.502V34.71L145.727 60.7359L115.26 33.5784Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M110.86 24.4126L143.696 147.866L177.887 24.4126C177.887 24.4126 158.478 46.8175 145.727 46.8175C132.976 46.8175 110.86 24.4126 110.86 24.4126Z\\" fill=\\"white\\"></path><path d=\\"M145.727 46.7046L133.541 64.4701L138.618 77.8225L135.459 118.446L143.696 147.753L152.949 119.69L148.887 77.1436L155.996 65.4885L145.727 46.7046Z\\" fill=\\"#D86856\\"></path></svg>"`;

exports[`Staff svg Basic staff 2 1`] = `"<svg viewBox=\\"0 0 298 172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><path d=\\"M115.105 21.0322V171.586H0.550049C0.550049 171.586 13.4119 69.9741 23.7495 53.139C34.2073 36.304 115.105 21.0322 115.105 21.0322Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M297.094 171.585H179.294V21.2725C187.708 22.956 259.35 37.6265 271.49 57.2273C284.232 78.0306 297.094 171.585 297.094 171.585Z\\" fill=\\"#4E3A2F\\"></path><path d=\\"M115.105 80.4354H179.294V0.228516H115.105V80.4354Z\\" fill=\\"#F1CBB9\\"></path><path d=\\"M115.105 44.3604C123.158 69.2522 171.961 69.2522 179.174 44.3604V171.585H115.105V44.3604Z\\" fill=\\"white\\"></path></svg>"`;

exports[`Staff svg Empty 1`] = `null`;
